dbt_platform_helper/providers/platform_config_schema.py

```diff
@@ -434,6 +436,22 @@ def valid_s3_bucket_name(name: str):
 
         return True
 
+    @staticmethod
+    def __datadog_schema() -> dict:
+        return {
+            "type": "datadog",
+            Optional("environments"): {
+                PlatformConfigSchema.__valid_environment_name(): {
+                    Optional("team_name"): str,
+                    Optional("contact_name"): str,
+                    Optional("contact_email"): str,
+                    Optional("repository"): str,
+                    Optional("docs"): str,
+                    Optional("services_to_monitor"): list,
```

Comment:
If the services are running off different codebases, would there be multiple Datadog extensions?

Reply:
Actually we can remove `Optional("repository"): str,` from here as we pull that in from the ci-image builder.
https://github.com/uktrade/ci-image-builder/blob/a63976b13865f2dbcade669590531710f1565adf/image_builder/pack.py#L152

Reply:
Every repository should have a README, even if it links out to documentation elsewhere.

Would is be fair to suggest that we don't need the docs option too?